app/components/Application/ApplicationDetailsCardItemCustomFields.tsx

```diff
@@ -49,11 +49,11 @@ const customFields = (
           return (
             <>
               <span id={id && `${id}-diffFrom`} className="diffFrom">
-                {prevValue ? prevValue : <i>[No Data Entered]</i>}
+                {prevValue ?? <i>[No Data Entered]</i>}
               </span>
               &nbsp;---&gt;&nbsp;
               <span id={id && `${id}-diffTo`} className="diffTo">
-                {currentValue ? currentValue : <i>[No Data Entered]</i>}
+                {currentValue ?? <i>[No Data Entered]</i>}
```

Comment:
This stuck out as a weird/unfamiliar syntax to me, but [it does seem to be quite useful](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#Assigning_a_default_value_to_a_variable).

Reply:
This and [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) were introduced in Typescript at some point last year. I'm happy it made its way to javascript